src/main/java/competition/operator_interface/OperatorCommandMap.java

```diff
@@ -81,4 +82,13 @@ public void setupClimbCommands(OperatorInterface oi, DynamicClimbCommand dynamic
         oi.operatorGamepad.getifAvailable(XboxButton.Back).whileHeld(dynamicClimb);
     }
 
+    @Inject
+    public void setUpHoodCommands(OperatorInterface oi, HoodSubsystem hood){
+        Command slowlyExtend = new InstantCommand(() -> hood.slowlyExtend());
```

Comment:
I'm not as familiar with the new `InstantCommand`, but my understanding is it ends after 1 execute by default? In that case the while held will keep starting the command over and over again? Might it be better to have a proper Command class that could keep running continuously? 

Can you double check me @JohnGilb